playbooks/common/openshift-cluster/openshift_logging.yml

```diff
@@ -1,14 +1,35 @@
 ---
-- name: OpenShift Aggregated Logging
+- name: Create Hosted Resources - logging
   hosts: oo_first_master
+  pre_tasks:
+  - set_fact:
+      logging_hostname: "{{ openshift_hosted_logging_hostname | default('kibana.' ~ (openshift_master_default_subdomain | default('router.default.svc.cluster.local', true))) }}"
```

Comment:
@ewolinetz, most of these could move into the logging role as defaults.

Not for this PR